### DISCUSSION TITLE
Undefined constant "FTP_BINARY"

### DISCUSSION THREAD

**Discussion by jesperbjoernandersen** [mlocati/docker-php-extension-installer#932] [Q&A]
I'm getting this error: Undefined constant "FTP_BINARY" {"exception":"[object] (Error(code: 0): Undefined constant \"FTP_BINARY\"

I'm running a Laravel application on my docker image of php:8.2-apache. I've already added "RUN install-php-extensions ftp" to my docker file. Any ideas?

**Comment by mlocati** [comment]
Are you installing the ftp PHP extension after starting apache?
